From 527149044bd9ad829a841ad770dbad688d01f2cc Mon Sep 17 00:00:00 2001
From: Dietmar Eggemann <dietmar.eggemann@arm.com>
Date: Fri, 25 Sep 2015 17:15:11 +0100
Subject: [PATCH 41/92] arm64: Wire frequency-invariant load-tracking support
 up to the scheduler

Define arch_scale_freq_capacity to use the arch-specific function
scale_freq_capacity().

Cc: Catalin Marinas <catalin.marinas@arm.com>
Cc: Will Deacon <will.deacon@arm.com>
Signed-off-by: Dietmar Eggemann <dietmar.eggemann@arm.com>
(cherry picked from commit 3cf6bce5357e08c7876df57c923aa27bf864c00a)
Signed-off-by: Gaku Inami <gaku.inami.xw@bp.renesas.com>
---
 arch/arm64/include/asm/topology.h | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/include/asm/topology.h b/arch/arm64/include/asm/topology.h
index 5d576eb..0f2c4dc 100644
--- a/arch/arm64/include/asm/topology.h
+++ b/arch/arm64/include/asm/topology.h
@@ -36,6 +36,13 @@ struct cpu_topology {
 #define arch_scale_cpu_capacity scale_cpu_capacity
 extern unsigned long scale_cpu_capacity(struct sched_domain *sd, int cpu);
 
+#ifdef CONFIG_CPU_FREQ
+
+#define arch_scale_freq_capacity scale_freq_capacity
+extern unsigned long scale_freq_capacity(struct sched_domain *sd, int cpu);
+
+#endif /* CONFIG_CPU_FREQ */
+
 #include <asm-generic/topology.h>
 
 #endif /* _ASM_ARM_TOPOLOGY_H */
-- 
1.9.1

